CHANGELOG.md

```diff
@@ -2,15 +2,27 @@
 
 ## Unreleased
 
-### Changes
+### Features
 
 - Introduce a `getMe()` method to retrieve the currently logged in user
   from OSIAM 3.0
-- The `BasicUser` class and the method `getCurrentUserBasic()` have been deprecated.
+
+### Changes
+
+- Remove `set` prefixes from single-valued attributes in the resource builders
 
 ### Fixes
 
+- Resources could be changed by using the copy-of constructor of the builders, though they were supposed
+  to be immutable.
 - A `400 BAD REQUEST` response creates a `BadRequestException` instead of the appropriate `ConflictException` 
```

Comment:
You tell me. After all, you wrote this line: https://github.com/osiam/connector4java/commit/709cdefd116ebf4bff798f58ec006021ca1edf2f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7 :)

But seriously, I can add another commit, that fixes this issue.